pymc/tests/test_step.py

```diff
@@ -1314,8 +1342,9 @@ def perform(self, node, inputs, outputs):
                     (nchains, ndraws * nsub)
                 )
                 Q_2_1 = np.concatenate(trace.get_sampler_stats("Q_2_1")).reshape((nchains, ndraws))
-                assert Q_1_0.mean(axis=1) == 0.0
-                assert Q_2_1.mean(axis=1) == 0.0
+                # This used to be a scrict zero equality!
+                assert np.isclose(Q_1_0.mean(axis=1), 0.0, atol=1e-4)
+                assert np.isclose(Q_2_1.mean(axis=1), 0.0, atol=1e-4)
```

Comment:
I don't know who is maintaining the MLDA step method, but I had to change this strict equality... Is that okay?

Reply:
Nobody is actively maintaining it. Also, didn't we want to move it to `pymc-experimental`?

Reply:
I think it is still maintained as it was ported by the original team to V4